src/main/java/org/springframework/cloud/scheduler/spi/kubernetes/EntryPointStyle.java

```diff
@@ -38,5 +42,29 @@
 	 * SPRING_APPLICATION_JSON environment variable. Command line arguments will be passed
 	 * as-is.
 	 */
-	boot
+	boot;
+
+	/**
+	 * Converts the string of the provided entry point style to the appropriate enum value
+	 * using {@link RelaxedNames}. Defaults to {@link EntryPointStyle#exec} if no matching
+	 * entry point style is found.
+	 *
+	 * @param entryPointStyle the entry point style to use
+	 * @return the converted {@link EntryPointStyle}
+	 */
+	public static EntryPointStyle relaxedValueOf(String entryPointStyle) {
+		for (EntryPointStyle candidate : EnumSet.allOf(EntryPointStyle.class)) {
+			for (String relaxedName : new RelaxedNames(candidate.name())) {
```

Comment:
RelaxedNames is not available in Boot 2.0.x :-(.

Reply:
we do the same thing in deployer, so i suppose lets just wait to deal with it and anything else when we bump to 2.0